petpals/messaging/routes.py

```diff
@@ -0,0 +1,193 @@
+from re import template
+from flask import Blueprint, redirect, render_template, url_for, request
+from flask_login import current_user, login_required
+from sqlalchemy import func, null, distinct
+from petpals import db, socket
+from petpals.models import Messages, User
+from datetime import datetime
+import time
+
+router = Blueprint(
+    'message_router',
+    __name__,
+    url_prefix='/messages',
+    template_folder='templates',
+    static_folder="static",
+)
+
+current_message = None
```

Comment:
I don't believe this is being used and should be removed.

Reply:
Done!